server/src/main/java/com/linecorp/centraldogma/server/command/StandaloneCommandExecutor.java

```diff
@@ -249,33 +254,35 @@ protected void doStop(@Nullable Runnable onReleaseLeadership) {
         }, repositoryWorker);
     }
 
-    private CompletableFuture<Revision> push(PushCommand c) {
+    private CompletableFuture<CommitResult> push(AbstractPushCommand<?> c, boolean applyPreviewDiff) {
```

Comment:
Not sure this is better. 😅
```suggestion
    private CompletableFuture<CommitResult> push(AbstractPushCommand<?> c, boolean normalizing) {
```